analyze.py

```diff
@@ -70,7 +69,7 @@ def is_fubar(results):
             has_wrong_pubkey = True
             logging.debug(conn['pubkey'][0] + ' is a fubar pubkey size')
             fubar = True
-        if ec_kex and pubkey_bits < min_ec_size:
+        if ec_kex and pubkey_bits < 160:
```

Comment:
that should be 256: https://wiki.mozilla.org/Security/Server_Side_TLS#Old_backward_compatibility